src/Utilities/useRemediationsList.js

```diff
@@ -0,0 +1,30 @@
+import { useAxiosWithPlatformInterceptors } from '@redhat-cloud-services/frontend-components-utilities/interceptors';
+const { API_BASE } = require('../config');
+import { useState, useEffect, useRef } from 'react';
+
+export const useRemediationsList = (remediation) => {
+  const axios = useAxiosWithPlatformInterceptors();
+  const [remediationsList, setRemediationsList] = useState();
+  const mounted = useRef(false);
+
+  useEffect(() => {
+    mounted.current = true;
+    const fetchData = async () => {
+      try {
+        const nameList = await axios.get(
+          `${API_BASE}/remediations/?fields[data]=${remediation.name}`
```

Comment:
This should just be `${API_BASE}/remediations/?fields[data]=${remediation.name}`.  `fields[data]=name` means just return the name field (and nothing else).

Reply:
ahh i see the difference now, ignore

Reply:
Yeah, I hit 'save' prematurely 😬.  I fixed my comment...

Reply:
Change is up!